docs/_includes/toc.md

```diff
@@ -0,0 +1,121 @@
+
+<details>
```

Comment:
Can we make each the toolbar not collapsed by default? Let's keep the sub-elements in User Guide collapsed, as they would take a lot of space. Basically looking like this:
<img width="397" alt="image" src="https://github.com/nasa/fprime/assets/49786685/a99ae801-e877-4528-af1c-fee7efe5d140">

Reply:
Question on this one: could we just make those top-level sections (Tutorials, User Guide, Design and Philosophy) not collapsible? That would be a faster change. And like you said, we'd still keep the sub-elements under User Guide collapsible. Or do you want to keep those top-level sections collapsible, but open by default?

Reply:
Yes that would actually probably be better. Let's also make them a little bigger/bold ?

Reply:
Please check out my latest prototype and let me know your thoughts--if you like it, I'll make a commit to this PR :+1: 
https://chr1st1ansears.github.io/fprime/